client/src/entrypoints/documents/document-chooser-modal.js

```diff
@@ -81,14 +84,12 @@ DOCUMENT_CHOOSER_MODAL_ONLOAD_HANDLERS = {
       });
 
       $('a.upload-one-now').on('click', function (e) {
-        // Set current collection ID at upload form tab
+        // Set current collection ID at upload form
         const collectionId = $('#collection_chooser_collection_id').val();
         if (collectionId) {
           $('#id_document-chooser-upload-collection').val(collectionId);
         }
 
-        // Select upload form tab
-        $('a[href="#upload"]').tab('show');
```

Comment:
Same question here, I see we’re removing this but it’s unclear to me what we’re replacing it with. The current approach doesn’t seem to work.

Reply:
I've fixed this now inside of wagtaildocs/includes/results.html. It now works similar to the button on the task chooser modal "Upload one now" button